google/cloud/bigquery/v2/minimal/internal/job_request_test.cc

```diff
@@ -1027,18 +1014,6 @@ TEST(PostQueryRequestTest, Success) {
   EXPECT_EQ(expected, *actual);
 }
 
-TEST(PostQueryRequestTest, MissingQuery) {
-  PostQueryRequest request("1234");
-
-  Options opts;
-  opts.set<EndpointOption>("bigquery.googleapis.com");
-  internal::OptionsSpan span(opts);
-
-  auto actual = BuildRestRequest(request);
-  EXPECT_THAT(actual, StatusIs(StatusCode::kInvalidArgument,
-                               HasSubstr("Missing required query field")));
```

Comment:
`StatusIs` and `HasSubstr` are now unused.  Please remove their using declarations.

Reply:
Done.